editoast/src/schema/speed_section.rs

```diff
@@ -58,9 +56,9 @@ pub struct SpeedSectionExtensions {
 #[derive(Debug, Default, Clone, Deserialize, Serialize, PartialEq)]
 #[serde(deny_unknown_fields)]
 pub struct SpeedSectionPslSncfExtension {
-    announcement: Vec<Sign>,
-    z: Sign,
-    r: Vec<Sign>,
+    pub announcement: Vec<Sign>,
```

Comment:
Are you sure it's ok to make `pub` these fields? Another possible solution would be to implement functions on `SpeedSectionPslSncfExtension `. I see 2 different ways of doing this:

```rust
impl SpeedSectionPslSncfExtension  {
  fn announcement(&self) -> &Vec<Sign> { /* ... */ }
  fn z(&self) -> &Sign { /* ... */ }
  fn r(&self) -> &Vec<Sign> { /* ... */ }
}
```

or maybe just what you need with the following

```rust
impl SpeedSectionPslSncfExtension {
  fn get_track_referenced_id(&self) -> impl Iterator<&String> { /* ... */ }
}
```

Reply:
ok, i will add getter